Makefile

```diff
@@ -69,13 +69,13 @@ compile-locale:
 	cd src/sentry && sentry django compilemessages
 
 install-transifex:
-	$(PIP) install transifex-client
+	[ -f ./tx ] || curl -o- https://raw.githubusercontent.com/transifex/cli/master/install.sh | bash
```

Comment:
curl bash makes me nervous -- I would prefer we control the script which does this download if possible -- looks like it mostly pulls from here https://github.com/transifex/cli/releases

we should download from there and checksum -- we can get away with only supporting (linux, mac), (x86_64, arm64) probably -- I'd recommend a small python similar to setup-volta: https://github.com/getsentry/action-setup-volta/blob/main/bin/setup-volta (can probably put this in `tools/`)